lab-yunjoo/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lab-yunjoo",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "bcrypt": "^0.8.6",
+    "body-parser": "^1.15.1",
+    "debug": "^2.2.0",
```

Comment:
Debug is a dependency, not a devDependency, because even though it's for us as developers, if you go to production and it isn't installed, all your code will break.